woocommerce/wc-admin
# Build: Move `lib/csv` to packages

Move the `lib/csv` utilities to a lerna package. This will now be `@woocommerce/csv`. There should be no functionality change.

To test:

- Build the project, make sure it succeeds.
- Check that downloading report tables still works.
